env.go

```diff
@@ -207,19 +208,24 @@ type ReadinessProbe interface {
 	Ready(runnable Runnable) (err error)
 }
 
-// HTTPReadinessProbe checks readiness by making HTTP call and checking for expected HTTP status code.
+// HTTPReadinessProbe checks readiness by making HTTP or HTTPS call and checking for expected HTTP/HTTPS status code.
 type HTTPReadinessProbe struct {
 	portName                 string
 	path                     string
+	scheme                   string
 	expectedStatusRangeStart int
```

Comment:
How about making this a bool? For now I cannot imagine scenario where we would need to specify more schemas, other than choosing between either HTTP or HTTPS.

Reply:
I think it is better to align with kubernetes - https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#http-probes

Reply:
That's a fair point, agreed then :+1: Would be still nicer to have this as a typed string with either `HTTP` or `HTTPS` option, but that's more of a nit.